### ISSUE TITLE
After applying resize, "Cancel" button is overlapping with data of select a resource dialog: A11y_Azure Tools Storage Explorer_Open Connect dialog_Resize

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/AzureStorageExplorer#8372]
### Preflight Checklist

- [x] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [x] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [x] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.36.2

### Architecture

x64

### Platform

Windows

### OS Version

Windows11

## GitHub Tags:
#A11yMAS; #A11yTCS; #Win11; #DesktopApp; #A11ySev2; #BM_AzureToolsStorageExplorer_Win32_Dec2024; #Azure Tools Storage Explorer; #Resize; #WCAG1.4.4; #FTP; 

## Environment Details:
Application: Microsoft azure storage explorer
Version: 1.36.2
OS: Windows 11 Enterprise 24H2
Build: 26100.2605

### Prerequisite to apply resize:

Change the display resolution to 1280*768 and drag the text size (System setting > Accessibility >Text size) slider to 200%

**Alternative method:**
Change the display resolution to 1366*768 and apply advanced scaling to 200.

## Repro Steps

1. Launch the Azure Tools Storage Explorer.
2. TAB till "Open Connect Dialog" present in left in left navigation and press ENTER key.
3. "Select a resource" dialog will open.
4. Apply resize
5. Observe that after applying resize, "Cancel" button is overlapping with data of select a resource dialog.

## Actual Experience

After applying resize, "Cancel" button is overlapping with data of select a resource dialog.
**Note:** Issue seen throughout the application with below dialogs

1. Upload file or folder dialog for blob
2. Add or view message dialog for Queue
3. Upload VHD dialog for disk
4. share access signature dialog

## Expected Experience

After applying resize, "Cancel" button should not overlap with data of select a resource dialog.

## User impact
Low visibility user who uses application after applying resize will not be able to see all data because after applying resize, "Cancel" button is overlapping with data of select a resource dialog.

## Attachment
![Image](https://github.com/user-attachments/assets/6b424ac1-c2e4-4029-880f-bb9219065327)

Alternative method:

Similar issue observed with other dialogs

![Image](https://github.com/user-attachments/assets/99dc2828-aa4d-4b2a-8bee-bcd043cb2a03)

**Comment by kapilvaishna** [comment]
@craxal please find attachment below
![Image](https://github.com/user-attachments/assets/119f783a-01b3-431f-b12a-787a1f76c2f6)

**Comment by craxal** [comment]
Seems like a combination of Windows scaling and text enlarging was required. Looks like the problem comes from the dialog content element's height being set to 100%. With Windows scaling and text sizing, the content element grows to a height larger than the dialog window. But because its height is set to 100%, it kind of messes up the layout calculations, which is why the dialog buttons are drawn in a very awkward position when you scroll down.

The fix is very simple. There's no reason the dialog content height needs to be set explicitly, since the containing element will automatically add the scroll bar if the window is too small. This corrects the layout calculation to use the actual height the dialog content and places the dialog buttons correctly.

**Comment by craxal** [comment]
Fix has been applied to most dialogs. A few more will require updates to additional extensions/packages.

@kapilvaishna Can you please specify which dialogs you have this issue?

**Comment by kapilvaishna** [comment]
@craxal  issue is observed with below dialogs also 

1. Upload file or folder dialog for blob
2. Add or view message dialog for Queue
3. Upload VHD dialog for disk
4. share access signature dialog

**Comment by craxal** [comment]
@kapilvaishna Fix is ready for validation.

**Comment by kapilvaishna** [comment]
@craxal Main issue and issue with share access signature dialog is not repro bug issue is repro with below dialogs. PFA for reference.

1. Upload file or folder dialog for blob
2. Add or view message dialog for Queue
3. Upload VHD dialog for disk


Repro attachment
![Image](https://github.com/user-attachments/assets/ef7e214d-5c05-4729-a253-362cefe29f62)

Fixed attachment
![Image](https://github.com/user-attachments/assets/077f2e97-2ec8-47c8-81e9-ca35d1e9fdd4)

**Comment by craxal** [comment]
@kapilvaishna Sorry, I forgot to send you a link to the new build. The build number you should have should be at least 20250128.2 or later. In the future, can you please report the build number that you test on?

**Comment by kapilvaishna** [comment]
Issue fixed for below to window

1. Add or view message dialog for Queue
2. Upload VHD dialog for disk


Fixed attachment 

![Image](https://github.com/user-attachments/assets/bfd6070e-0e1a-4dd0-bb50-a047548bbcea)

**Comment by craxal** [comment]
@kapilvaishna Let's close this bug, since it initially referred to the Connect dialog. We can open a new bug for the upload dialog.

**Comment by kapilvaishna** [comment]
As issue is only with 'Upload file or folder' window for blob all other issues are fixed so closing this bug and logging a new bug for that.
